Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt-get install -y \
     php7.0-pdo-mysql
 
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
-RUN npm install -g yarn
+RUN npm install -g yarn m2-builder@1.0.6
```

Comment:
@dcabrejas can you ensure this is up to date with latest version seen here https://www.npmjs.com/package/m2-builder

Reply:
1.2.0 in this case

Reply:
is there not a ^1.0 option we can do with NPM ?

Reply:
`m2-builder@1`